source/tools/glkernel-cli/CMakeLists.txt

```diff
@@ -0,0 +1,153 @@
+
+#
+# External dependencies
+#
+
+find_package(cppassist REQUIRED)
+find_package(cppexpose REQUIRED)
+find_package(PNG REQUIRED)
+
+#
+# Executable name and options
+#
+
+# Target name
+set(target glkernel-cli)
+
+
+#
+# Sources
+#
+
+set(headers
+    AbstractKernelExporter.h
+    JsonExporter.h
+    KernelGenerator.h
+    KernelObject.h
+    JSInterface.h
+    JsonImporter.h
+    PngExporter.h
+    helper.h
+    )
```

Comment:
We choose to indent the closing brace as much as the line containing the opening one. In this case: no identation.